tests/decode_images.rs

```diff
@@ -183,6 +187,43 @@ fn issue_69() {
     test_image_sum_u16("issue_69_packbits.tiff", ColorType::Gray(16), 1015486);
 }
 
+/// Relies on the "generate_new_hash_for_scanimage_metadata_parser" 'test' to
+/// generate the correct hash value to be compared.
+#[test]
+fn test_si_header_metadata_parser() {
+    let filename = "ScanImageBigTIFFv57.tif";
+    let path = PathBuf::from(TEST_IMAGE_DIR).join(BIGTIFF_DIR).join(filename);
+    let file = File::open(path).unwrap();
+    let decoder = Decoder::new(file).unwrap().with_scanimage();
+    let meta = decoder.scanimage_metadata().unwrap();
+    let mut hasher_roi = DefaultHasher::new();
+    let mut hasher_frame = DefaultHasher::new();
```

Comment:
The default hasher is not deterministic between runs, by design. Use a crc such as `crc32fast` instead.